src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
+        Movie currentMovie = get(movie.getId());
+        movieDao.update(movie);
+        return currentMovie.equals(get(movie.getId()));
```

Comment:
The logic in the `update` method is inefficient. Instead of comparing the current and updated `Movie` objects, consider returning `true` after a successful update operation.